cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -92,10 +92,12 @@ public void execute() {
 
                     String defaultOrg = guessOrgName();
 
-                    // Get org name.
-                    out.print("Organization name: (" + defaultOrg + ") ");
-                    String orgName = scanner.nextLine().trim();
-                    manifest.setName(orgName.isEmpty() ? defaultOrg : orgName);
+                    String orgName;
+                    do {
+                        out.print("Organization name: (" + defaultOrg + ") ");
+                        orgName = scanner.nextLine().trim();
+                        manifest.setName(orgName.isEmpty() ? defaultOrg : orgName);
```

Comment:
Do we need to set it here

Reply:
Moved the org-name setting outside the loop with af18af2